oslabs-beta/axon
# Regex update to work on diff systems and create btn to call main algo

## **Types of Changes**
## **Purpose**
- to recognize regex line breaks for different operating systems
## **Approach**
- included /r?/n to tempEndpointData.js 
## **How to Test**
- Create a SupeTtest file in both Mac and Windows systems (check if test was correctly generated)
## **Screenshot(s)**
![GUIcapture](https://user-images.githubusercontent.com/59153906/116792200-02830000-aa74-11eb-99fb-e6b86b9f2fdf.PNG)
